src/main/java/org/reaktivity/nukleus/kafka/internal/KafkaConfiguration.java

```diff
@@ -19,9 +19,27 @@
 
 public class KafkaConfiguration extends Configuration
 {
+    public static final String PROPERTY_BROKER_AUTO_RECONNECT = "kafka.auto.reconnect";
+
+    public static final boolean DEFAULT_BROKER_AUTO_RECONNECT  = true;
+
     public KafkaConfiguration(
         Configuration config)
     {
         super(config);
     }
+
+    // TODO: do partial decoding and acknowledging of Kafka responses and partial encoding
+    //       to avoid the need for this method
+    public int  maximumUnacknowledgedBytes()
+    {
+        return getInteger("reaktor.memory.block.capacity", 8 * 1024);
```

Comment:
Will do. The intended meaning is that TCP nukleus will guarantee to send us at least that amount of bytes  without us acknowledging any of it. I will set it to 64k for now by default since  I believe that's what TCP will  default to.